examples/encrypt.rs

```diff
@@ -47,20 +45,19 @@ fn generate_new(
 }
 
 fn main() {
-    let policy = policy().expect("cannot generate policy");
     let ap = AccessPolicy::parse("Department::FIN && Security Level::Top Secret").unwrap();
 
     let cc = Covercrypt::default();
     let (mut msk, _) = cc.setup().expect("cannot generate master keys");
     let mpk = cc
-        .update_master_keys(&policy, &mut msk)
+        .update_master_keys(&mut msk)
```

Comment:
This function should be replaced by a `cc.gen_mpk(&msk)` function since the `msk` should already be up-to-date.